### PR TITLE
httplz: init at 1.5.0

### PR DESCRIPTION
**Pull request by bbigras**
###### Motivation for this change

Note that the project's name is "http" and it provides 2 identical binaries ("http" and "httplz").

"http" collides with the very popular httpie's "http" so I named the package "httplz" and I only build the "httplz" binary.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -1689,6 +1689,8 @@ in
 
   nx-libs = callPackage ../tools/X11/nx-libs { };
 
+  httplz = callPackage ../tools/networking/httplz { };
```
Comment (JohnAZoidberg) [inline pkgs/top-level/all-packages.nix]: Put in the correct spot, alphabetically, please ;)

**Comment by JohnAZoidberg** [comment]
> "http" collides with the very popular httpie's "http" so I named the package "httplz" and I only build the "httplz" binary.

Good catch! Well we could include the `http` binary and change the priorities so that httpie overrides this. But I think that adds an unnecessary maintenance burden for us. A user who wants the shorter name can just add an alias for themself.

Built and tested it - works fine :)

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (bbigras) [reply pkgs/top-level/all-packages.nix]: Done

**Review by JohnAZoidberg — APPROVED** [review]

**Comment by bbigras** [comment]
Thanks!

**Pull request merged**